action.yml

```diff
@@ -58,5 +62,5 @@ runs:
         docker pull ${{ inputs.security_control }}
         JSON_GITHUB_EVENT='${{ toJSON(github) }}'
         B64_ENCODED_GITHUB_EVENT=$(echo $JSON_GITHUB_EVENT | base64)
-        docker run --rm -e CONFIG_FILE_PATH=/.jit/jit.yml -e GITHUB_EVENT="$B64_ENCODED_GITHUB_EVENT" -e SOURCE_CODE_FOLDER=/code -e SECURITY_CONTROL_OUTPUT_FILE=${{ inputs.security_control_output_file }} -v $(pwd)/.jit:/.jit -e FAIL_ON_FINDINGS=${{ inputs.fail_on_findings }} -v $(pwd)/code:/code ${{ inputs.security_control }} -- ${{ inputs.security_control_args }}
+        docker run --rm -e CONFIG_FILE_PATH=/.jit/jit.yml ${{ inputs.container_args }} -e GITHUB_EVENT="$B64_ENCODED_GITHUB_EVENT" -e SOURCE_CODE_FOLDER=/code -e SECURITY_CONTROL_OUTPUT_FILE=${{ inputs.security_control_output_file }} -v $(pwd)/.jit:/.jit -e FAIL_ON_FINDINGS=${{ inputs.fail_on_findings }} -v $(pwd)/code:/code ${{ inputs.security_control }} -- ${{ inputs.security_control_args }}
```

Comment:
Thinking about the near future, secrets will probably be a part of the payload and not an env variable.
with that, will we really need this field?

Reply:
i think so, there are other possible args we want to pass, i.e. mounts, permissions...

Reply:
ok